packages/cli/src/common/filenames.ts

```diff
@@ -3,3 +3,32 @@ import * as inflected from 'inflected'
 export function classNameToFileName(name: string): string {
   return inflected.dasherize(inflected.underscore(name))
 }
+
+export async function checkResourceNameIsValid(name: string): Promise<void> {
```

Comment:
one question, Why is this function async? I don't see any async call inside the method

Reply:
Hey, @rdiaz82 yeap, looks like it is not required, I only used it to do fancy `.then/.catch` in test file 😸